api.ts

```diff
@@ -153,9 +153,16 @@ export interface SourceFileConfiguration {
     readonly defines: string[];
 
     /**
-     * The compiler to emulate.
+     * The platform, compiler, and architecture variant to emulate.
      */
-    readonly intelliSenseMode?: "msvc-x86" | "msvc-x64" | "msvc-arm" | "msvc-arm64" |
+    readonly intelliSenseMode?: "linux-clang-x86" | "linux-clang-x64" | "linux-clang-arm" | "linux-clang-arm64" |

```

Comment:
Should this be formatted better to make it easier to read/compare? Maybe all the platforms on a singe line?

Reply:
I think it is fine this way.